src/app/zones/locales/en-us/index.yaml

```diff
@@ -74,15 +74,22 @@ zones:
     create: 'Create Zone'
   form:
     exit: 'Exit'
+    title: 'Create & connect Zone'
+    description: ' '
+    facts: ['One', 'Two', 'Three', 'Four']
```

Comment:
Not sure if you meant to leave these in or not, will approve anyway, but just incase you didn't notice these

Reply:
Thanks, removed those.